src/styles/blocks/header.scss

```diff
@@ -0,0 +1,12 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  font-size: 12px;
+  font-weight: 500;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    align-self: center;
+  }
+}
```

Comment:
add align-items: center to the flex container and you won't need to add align-self to logo